gadopt/stokes_integrators.py

```diff
@@ -60,15 +61,15 @@ def create_stokes_nullspace(Z, closed=True, rotational=False, translations=None)
     """
     X = fd.SpatialCoordinate(Z.mesh())
     dim = len(X)
-    V, W = Z.subfunctions
+    stokes_funcspace = Z.subfunctions
```

Comment:
Out of curiosity, why `funcspace`? I think this is slightly confusing, given that `subfunctions` returns a list of `Functions` and that Firedrake also defines `FunctionSpace`.

Reply:
So I am pretty sure they are returning functionspaces, because Z is the mixed function space. If you look [here](https://www.firedrakeproject.org/firedrake.html#firedrake.functionspaceimpl.FunctionSpace) Functionspace does have an attribute called subfunctions which gives you the component function spaces. I agree it seems like a confusing naming convention

Reply:
My bad. I did not know that was a thing. Thanks, Will.

Reply:
changed now!